certified-connectors/Scancloud/readme.md

```diff
@@ -0,0 +1,30 @@
+
+## Scancloud Connector
+Scancloud provides a powerful platform for pluggable security using Scancloud plugins.
```

Comment:
@Ajaatsecplugs please provide some information about the main purpose and value offered by your connector.

Reply:
Added